docs/metrics.md

```diff
@@ -124,39 +109,44 @@ Services.telemetry.scalarSet(
 	"category.scalar_name", value
 );
 ```
-e.g. `lockboxV1.datastoreCount` for the scalar name.
+e.g. `lockbox-addon.loginCount` for the scalar name.
 
 We can also use `scalarAdd` to increment a scalar value by some amount.
 
 ## Scalar Metrics
 
-These are the metrics we currently collect regarding the state of user datastores.
+These are the metrics we currently collect regarding the state of the user's login storage.
+
+- `loginCount` (SCALAR_TYPE_COUNT). Current count of the number of items in the user's login storage. If possible, this scalar should be set upon opening either the doorhanger or full extension interface.
+
+- `loginsAccessedAllTime` (SCALAR_TYPE_COUNT). Running total of logins accessed (copied, revealed) through the extension interface or doorhanger. Should never be reset.
```

Comment:
it's not clear to me, from the interface descriptions, how possible either this or `loginsAccessed` is from the Telemetry WebExtension API.  One of them definitely is, but it's not clear which; not clear how much effort it will take to have both.

Reply:
I'm pretty sure that `scalarAdd` increments a per-browsing session counter, so `loginsAccessedAllTime` would require saving the all-time count in webextension browser storage (not that hard to do).